### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Update readme

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ Start with these names for your branch depending on what your branch includes.
   - 'dependencies/'
   - 'documentation/'
   - 'enhancement/'
-  - 'feature/'
+  - feat/single-rights/issue-number
+  - feat/api-delegation/issue-number
+  - feat/general/issue-number       (When it's not related to any features)
```
Comment (allinox) [inline README.md:45]: Should these also be annoted with ''?

Comment (Albertlarsen) [reply README.md:45]: Hmm I don't know. Is the annotation necessary? I removed them on the other just now as a proposed solution

Comment (allinox) [reply README.md:45]: Idk, I just noticed the difference 😅
I checked how it looks when "rendered" now and it looks good I think

**Review by allinox — APPROVED** [review]
Awesome work, keeping the readme updated!
A golden star sticker for you! ⭐📘

**Pull request merged**